wire-schema/api/wire-schema.api

```diff
@@ -1203,16 +1203,18 @@ public final class com/squareup/wire/schema/internal/parser/ExtendElement {
 }
 
 public final class com/squareup/wire/schema/internal/parser/ExtensionsElement {
-	public fun <init> (Lcom/squareup/wire/schema/Location;Ljava/lang/String;Ljava/util/List;)V
-	public synthetic fun <init> (Lcom/squareup/wire/schema/Location;Ljava/lang/String;Ljava/util/List;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (Lcom/squareup/wire/schema/Location;Ljava/lang/String;Ljava/util/List;Ljava/util/List;)V
```

Comment:
Should we be concerned about breaking binary compat?

Reply:
I'll put that in Wire 5 so I would not say so.